paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodRowCheckmarkButtonScreenshotTest.kt

```diff
@@ -210,8 +211,8 @@ internal class PaymentMethodRowCheckmarkButtonScreenshotTest {
         Row(
             verticalAlignment = Alignment.CenterVertically,
             modifier = Modifier
-                .testTag(TEST_TAG_VIEW_MORE)
-                .padding(4.dp)
+                .padding(start = 4.dp, end = 0.dp, top = 4.dp, bottom = 4.dp)
+                .offset(x = 9.dp)
```

Comment:
These look the same as another location. We should find a way to dedupe as well as make them less magic.